core/rendered_connection.js

```diff
@@ -206,10 +206,12 @@ Blockly.RenderedConnection.prototype.highlight = function() {
   var x = this.x_ - xy.x;
   var y = this.y_ - xy.y;
   Blockly.Connection.highlightedPath_ = Blockly.utils.createSvgElement('path',
-      {'class': 'blocklyHighlightedConnectionPath',
-       'd': steps,
-       transform: 'translate(' + x + ',' + y + ')' +
-           (this.sourceBlock_.RTL ? ' scale(-1 1)' : '')},
+      {
+        'class': 'blocklyHighlightedConnectionPath',
+        'd': steps,
+        transform: 'translate(' + x + ',' + y + ')' +
+           (this.sourceBlock_.RTL ? ' scale(-1 1)' : '')
```

Comment:
One more space here.

Reply:
Done.  
Eslint isn't checking this because it doesn't care what you do as you add strings, as far as I can tell.